resources.md

```diff
@@ -37,12 +37,13 @@ The following client libraries can be used to run queries from several
 programming languages, and programs using the related platform:
 
 * [Trino JDBC driver](docs/current/client/jdbc.html) for Java/JVM
-* [Starburst ODBC driver](https://docs.starburst.io/data-consumer/clients/odbc.html) for Windows
```

Comment:
Are you sure it doesnt work for Windows with specific license setups?

Reply:
Change to https://www.magnitude.com/drivers/trino-odbc-jdbc and Magnitude ODBC driver

Reply:
Starburst ODBC driver doesn't support Trino, changed to Magnitude ODBC driver.